File: platform/Leetcode/src/words.py
# Python solution
def find_length_last(words):
    word_list = words.strip().split(' ')    
    return len(word_list[-1]) if len(word_list)>0 else 0

if __name__ == "__main__":
    print("Input a phrase below:")
    words = input()
    print("the length of last word is:", find_length_last(words))

File: platform/Leetcode/src/Words.java
import java.util.Scanner;

public class Words { 
    
    public static void main(String[] args) {
        System.out.println("Input a string of text:");
        Scanner input = new Scanner(System.in);
        int length = countLength(input.next());
        System.out.println("The length of last word is " + Integer.toString(length));
    }
    
    public static int countLength (String phrase) {
        String trimmedPhrase = phrase.trim();
        String[] splitPhrase = trimmedPhrase.split("\\s+");
        String lastWord = splitPhrase[splitPhrase.length - 1];
        if (!lastWord.equals("")) {
            return lastWord.length();
        } else {
            return 0;
        }
    }
    
}